[9.1] [SecuritySolution][Endpoint][ResponseActions] Response actions telemetry tests for third party agents (#225916)

kibanamachine opened the pull request:
# Backport

This will backport the following commits from `main` to `9.1`:
 - [[SecuritySolution][Endpoint][ResponseActions] Response actions telemetry tests for third party agents (#225916)](https://github.com/elastic/kibana/pull/225916)

<!--- Backport version: 9.6.6 -->

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--BACKPORT [{"author":{"name":"Ash","email":"1849116+ashokaditya@users.noreply.github.com"},"sourceCommit":{"committedDate":"2025-07-03T14:44:39Z","message":"[SecuritySolution][Endpoint][ResponseActions] Response actions telemetry tests for third party agents (#225916)\n\n## Summary\n\nThis is a follow up of https://github.com/elastic/kibana/pull/221518. \n\n- [x] Adds missing tests for action creation and action response events\nfor third party agents.\n- [x] Adds telemetry events for completed action responses for\ncrowdstrike agents.\n\n**Note:** The action creation tests in each agent type is redundant as\nthe action creation telemetry event is sent from a method in the base\naction client class, but tests are added here just to ensure that\ntelemetry events are not broken by any changes in the future for third\nparty agents.\n\n### Checklist\n\nCheck the PR satisfies following conditions. \n\nReviewers should verify this PR satisfies this list as well.\n\n- [ ] Any text added follows [EUI's writing\nguidelines](https://elastic.github.io/eui/#/guidelines/writing), uses\nsentence case text and includes [i18n\nsupport](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)\n- [ ]\n[Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html)\nwas added for features that require explanation or tutorials\n- [x] [Unit or functional\ntests](https://www.elastic.co/guide/en/kibana/master/development-tests.html)\nwere updated or added to match the most common scenarios\n- [ ] If a plugin configuration key changed, check if it needs to be\nallowlisted in the cloud and added to the [docker\nlist](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)\n- [ ] This was checked for breaking HTTP API changes, and any breaking\nchanges have been approved by the breaking-change committee. The\n`release_note:breaking` label should be applied in these situations.\n- [ ] [Flaky Test\nRunner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was\nused on any tests changed\n- [ ] The PR description includes the appropriate Release Notes section,\nand the correct `release_note:*` label is applied per the\n[guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)\n- [ ] Review the [backport\nguidelines](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?usp=sharing)\nand apply applicable `backport:*` labels.\n\n### Identify risks\n\nDoes this PR introduce any risks? For example, consider risks like hard\nto test bugs, performance regression, potential of data loss.\n\nDescribe the risk, its severity, and mitigation for each identified\nrisk. Invite stakeholders and evaluate how to proceed before merging.\n\n- [ ] [See some risk\nexamples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)\n- [ ] ...\n\n---------\n\nCo-authored-by: kibanamachine <42973632+kibanamachine@users.noreply.github.com>","sha":"11bb9235d6afb98bbd4147a7d2225fc6af1b2acc","branchLabelMapping":{"^v9.2.0$":"main","^v(\\d+).(\\d+).\\d+$":"$1.$2"}},"sourcePullRequest":{"labels":["release_note:skip","Team:Defend Workflows","OLM Sprint","backport:version","v9.1.0","v8.19.0","v9.2.0"],"title":"[SecuritySolution][Endpoint][ResponseActions] Response actions telemetry tests for third party agents","number":225916,"url":"https://github.com/elastic/kibana/pull/225916","mergeCommit":{"message":"[SecuritySolution][Endpoint][ResponseActions] Response actions telemetry tests for third party agents (#225916)\n\n## Summary\n\nThis is a follow up of https://github.com/elastic/kibana/pull/221518. \n\n- [x] Adds missing tests for action creation and action response events\nfor third party agents.\n- [x] Adds telemetry events for completed action responses for\ncrowdstrike agents.\n\n**Note:** The action creation tests in each agent type is redundant as\nthe action creation telemetry event is sent from a method in the base\naction client class, but tests are added here just to ensure that\ntelemetry events are not broken by any changes in the future for third\nparty agents.\n\n### Checklist\n\nCheck the PR satisfies following conditions. \n\nReviewers should verify this PR satisfies this list as well.\n\n- [ ] Any text added follows [EUI's writing\nguidelines](https://elastic.github.io/eui/#/guidelines/writing), uses\nsentence case text and includes [i18n\nsupport](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)\n- [ ]\n[Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html)\nwas added for features that require explanation or tutorials\n- [x] [Unit or functional\ntests](https://www.elastic.co/guide/en/kibana/master/development-tests.html)\nwere updated or added to match the most common scenarios\n- [ ] If a plugin configuration key changed, check if it needs to be\nallowlisted in the cloud and added to the [docker\nlist](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)\n- [ ] This was checked for breaking HTTP API changes, and any breaking\nchanges have been approved by the breaking-change committee. The\n`release_note:breaking` label should be applied in these situations.\n- [ ] [Flaky Test\nRunner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was\nused on any tests changed\n- [ ] The PR description includes the appropriate Release Notes section,\nand the correct `release_note:*` label is applied per the\n[guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)\n- [ ] Review the [backport\nguidelines](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?usp=sharing)\nand apply applicable `backport:*` labels.\n\n### Identify risks\n\nDoes this PR introduce any risks? For example, consider risks like hard\nto test bugs, performance regression, potential of data loss.\n\nDescribe the risk, its severity, and mitigation for each identified\nrisk. Invite stakeholders and evaluate how to proceed before merging.\n\n- [ ] [See some risk\nexamples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)\n- [ ] ...\n\n---------\n\nCo-authored-by: kibanamachine <42973632+kibanamachine@users.noreply.github.com>","sha":"11bb9235d6afb98bbd4147a7d2225fc6af1b2acc"}},"sourceBranch":"main","suggestedTargetBranches":["9.1","8.19"],"targetPullRequestStates":[{"branch":"9.1","label":"v9.1.0","branchLabelMappingKey":"^v(\\d+).(\\d+).\\d+$","isSourceBranch":false,"state":"NOT_CREATED"},{"branch":"8.19","label":"v8.19.0","branchLabelMappingKey":"^v(\\d+).(\\d+).\\d+$","isSourceBranch":false,"state":"NOT_CREATED"},{"branch":"main","label":"v9.2.0","branchLabelMappingKey":"^v9.2.0$","isSourceBranch":true,"state":"MERGED","url":"https://github.com/elastic/kibana/pull/225916","number":225916,"mergeCommit":{"message":"[SecuritySolution][Endpoint][ResponseActions] Response actions telemetry tests for third party agents (#225916)\n\n## Summary\n\nThis is a follow up of https://github.com/elastic/kibana/pull/221518. \n\n- [x] Adds missing tests for action creation and action response events\nfor third party agents.\n- [x] Adds telemetry events for completed action responses for\ncrowdstrike agents.\n\n**Note:** The action creation tests in each agent type is redundant as\nthe action creation telemetry event is sent from a method in the base\naction client class, but tests are added here just to ensure that\ntelemetry events are not broken by any changes in the future for third\nparty agents.\n\n### Checklist\n\nCheck the PR satisfies following conditions. \n\nReviewers should verify this PR satisfies this list as well.\n\n- [ ] Any text added follows [EUI's writing\nguidelines](https://elastic.github.io/eui/#/guidelines/writing), uses\nsentence case text and includes [i18n\nsupport](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)\n- [ ]\n[Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html)\nwas added for features that require explanation or tutorials\n- [x] [Unit or functional\ntests](https://www.elastic.co/guide/en/kibana/master/development-tests.html)\nwere updated or added to match the most common scenarios\n- [ ] If a plugin configuration key changed, check if it needs to be\nallowlisted in the cloud and added to the [docker\nlist](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)\n- [ ] This was checked for breaking HTTP API changes, and any breaking\nchanges have been approved by the breaking-change committee. The\n`release_note:breaking` label should be applied in these situations.\n- [ ] [Flaky Test\nRunner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was\nused on any tests changed\n- [ ] The PR description includes the appropriate Release Notes section,\nand the correct `release_note:*` label is applied per the\n[guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)\n- [ ] Review the [backport\nguidelines](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?usp=sharing)\nand apply applicable `backport:*` labels.\n\n### Identify risks\n\nDoes this PR introduce any risks? For example, consider risks like hard\nto test bugs, performance regression, potential of data loss.\n\nDescribe the risk, its severity, and mitigation for each identified\nrisk. Invite stakeholders and evaluate how to proceed before merging.\n\n- [ ] [See some risk\nexamples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)\n- [ ] ...\n\n---------\n\nCo-authored-by: kibanamachine <42973632+kibanamachine@users.noreply.github.com>","sha":"11bb9235d6afb98bbd4147a7d2225fc6af1b2acc"}}]}] BACKPORT-->

ashokaditya committed:
[SecuritySolution][Endpoint][ResponseActions] Response actions telemetry tests for third party agents (#225916)

## Summary

This is a follow up of https://github.com/elastic/kibana/pull/221518.

- [x] Adds missing tests for action creation and action response events
for third party agents.
- [x] Adds telemetry events for completed action responses for
crowdstrike agents.

**Note:** The action creation tests in each agent type is redundant as
the action creation telemetry event is sent from a method in the base
action client class, but tests are added here just to ensure that
telemetry events are not broken by any changes in the future for third
party agents.

### Checklist

Check the PR satisfies following conditions.

Reviewers should verify this PR satisfies this list as well.

- [ ] Any text added follows [EUI's writing
guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses
sentence case text and includes [i18n
support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)
- [ ]
[Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html)
was added for features that require explanation or tutorials
- [x] [Unit or functional
tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html)
were updated or added to match the most common scenarios
- [ ] If a plugin configuration key changed, check if it needs to be
allowlisted in the cloud and added to the [docker
list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)
- [ ] This was checked for breaking HTTP API changes, and any breaking
changes have been approved by the breaking-change committee. The
`release_note:breaking` label should be applied in these situations.
- [ ] [Flaky Test
Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was
used on any tests changed
- [ ] The PR description includes the appropriate Release Notes section,
and the correct `release_note:*` label is applied per the
[guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)
- [ ] Review the [backport
guidelines](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?usp=sharing)
and apply applicable `backport:*` labels.

### Identify risks

Does this PR introduce any risks? For example, consider risks like hard
to test bugs, performance regression, potential of data loss.

Describe the risk, its severity, and mitigation for each identified
risk. Invite stakeholders and evaluate how to proceed before merging.

- [ ] [See some risk
examples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)
- [ ] ...

---------

Co-authored-by: kibanamachine <42973632+kibanamachine@users.noreply.github.com>
(cherry picked from commit 11bb9235d6afb98bbd4147a7d2225fc6af1b2acc)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/315515)
* Commit: 68fb09d1c2a1ad5badfc6ebb8fce6aefd41c7387

### Failed CI Steps
* [Jest Tests #4](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d0ec-33af-4a2f-adfa-daf3424521fa)
* [Jest Tests #4](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d11c-93a3-487a-bcb4-a9544832e052)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d0ec-33af-4a2f-adfa-daf3424521fa) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315515/jobs/0197d0ec-33af-4a2f-adfa-daf3424521fa/artifacts/0197d11c-6270-4851-b910-b130e0417dd0) Jest Tests #<span></span>4 / CrowdstrikeActionsClient class #<span></span>runscript() should send action to Crowdstrike
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d11c-93a3-487a-bcb4-a9544832e052) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315515/jobs/0197d11c-93a3-487a-bcb4-a9544832e052/artifacts/0197d128-5023-4895-bfe8-84fa88cde735) Jest Tests #<span></span>4 / CrowdstrikeActionsClient class #<span></span>runscript() should send action to Crowdstrike
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d0ec-33af-4a2f-adfa-daf3424521fa) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315515/jobs/0197d0ec-33af-4a2f-adfa-daf3424521fa/artifacts/0197d11c-6270-46ad-93f4-5baed08425bd) Jest Tests #<span></span>4 / CrowdstrikeActionsClient class #<span></span>runscript() should write action request to endpoint indexes
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d11c-93a3-487a-bcb4-a9544832e052) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315515/jobs/0197d11c-93a3-487a-bcb4-a9544832e052/artifacts/0197d128-5023-474e-bf61-f5c88f0fbf53) Jest Tests #<span></span>4 / CrowdstrikeActionsClient class #<span></span>runscript() should write action request to endpoint indexes
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d0ec-33af-4a2f-adfa-daf3424521fa) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315515/jobs/0197d0ec-33af-4a2f-adfa-daf3424521fa/artifacts/0197d11c-6270-4f64-95d5-904fb758fbf5) Jest Tests #<span></span>4 / CrowdstrikeActionsClient class #<span></span>runscript() telemetry events should send `runscript` action response telemetry event for failed action
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d11c-93a3-487a-bcb4-a9544832e052) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315515/jobs/0197d11c-93a3-487a-bcb4-a9544832e052/artifacts/0197d128-5023-4026-b26b-233cc8a82398) Jest Tests #<span></span>4 / CrowdstrikeActionsClient class #<span></span>runscript() telemetry events should send `runscript` action response telemetry event for failed action
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d0ec-33af-4a2f-adfa-daf3424521fa) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315515/jobs/0197d0ec-33af-4a2f-adfa-daf3424521fa/artifacts/0197d11c-626f-4474-92b1-e50a0a4c7387) Jest Tests #<span></span>4 / CrowdstrikeActionsClient class #<span></span>runscript() telemetry events should send `runscript` action response telemetry event for successful action
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315515#0197d11c-93a3-487a-bcb4-a9544832e052) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315515/jobs/0197d11c-93a3-487a-bcb4-a9544832e052/artifacts/0197d128-5024-459c-9a6f-6b2678fe0072) Jest Tests #<span></span>4 / CrowdstrikeActionsClient class #<span></span>runscript() telemetry events should send `runscript` action response telemetry event for successful action

cc @ashokaditya

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/315515","number":315515,"commit":"68fb09d1c2a1ad5badfc6ebb8fce6aefd41c7387"}],"number":315515}
buildkite-pr-comment-->

ashokaditya committed:
Merge branch '9.1' into backport/9.1/pr-225916

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/315596)
* Commit: d3cbdd63c5a6246031d9385de89b111e49bfa0f9

### Failed CI Steps
* [Jest Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ca-ca9b-4f36-8d99-facc6d498d37)
* [FTR Configs #46](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ca-cac7-4e42-8885-5609dcc2e086)
* [FTR Configs #46](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ef-a196-468a-92af-5b01649dba7d)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ca-ca9b-4f36-8d99-facc6d498d37) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315596/jobs/0197d1ca-ca9b-4f36-8d99-facc6d498d37/artifacts/0197d1ee-b707-44fc-8066-104ac50e6a7d) Jest Tests #<span></span>1 / CrowdstrikeActionsClient class #<span></span>runscript() should send action to Crowdstrike
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ca-ca9b-4f36-8d99-facc6d498d37) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315596/jobs/0197d1ca-ca9b-4f36-8d99-facc6d498d37/artifacts/0197d1ee-b707-44d1-b815-471aa6bbc6e1) Jest Tests #<span></span>1 / CrowdstrikeActionsClient class #<span></span>runscript() should write action request to endpoint indexes
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ca-ca9b-4f36-8d99-facc6d498d37) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315596/jobs/0197d1ca-ca9b-4f36-8d99-facc6d498d37/artifacts/0197d1ee-b707-4247-bec9-60eba52012c2) Jest Tests #<span></span>1 / CrowdstrikeActionsClient class #<span></span>runscript() telemetry events should send `runscript` action response telemetry event for failed action
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ca-ca9b-4f36-8d99-facc6d498d37) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315596/jobs/0197d1ca-ca9b-4f36-8d99-facc6d498d37/artifacts/0197d1ee-b707-43ef-aec8-cb24562dd667) Jest Tests #<span></span>1 / CrowdstrikeActionsClient class #<span></span>runscript() telemetry events should send `runscript` action response telemetry event for successful action
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ca-cac7-4e42-8885-5609dcc2e086) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315596/jobs/0197d1ca-cac7-4e42-8885-5609dcc2e086/artifacts/0197d1ef-43a3-43a7-bf94-1538b963265a) FTR Configs #<span></span>46 / ObservabilityApp Observability alerts > Alerts table Filtering Autocompletion works
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315596#0197d1ef-a196-468a-92af-5b01649dba7d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315596/jobs/0197d1ef-a196-468a-92af-5b01649dba7d/artifacts/0197d1f9-5700-4bd2-96a3-044bc2f7a82f) FTR Configs #<span></span>46 / ObservabilityApp Observability alerts > Alerts table Filtering Autocompletion works

### History
* :broken_heart: [Build #315515](https://buildkite.com/elastic/kibana-pull-request/builds/315515) failed 68fb09d1c2a1ad5badfc6ebb8fce6aefd41c7387

cc @ashokaditya

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/315596","number":315596,"commit":"d3cbdd63c5a6246031d9385de89b111e49bfa0f9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/315515","number":315515,"commit":"68fb09d1c2a1ad5badfc6ebb8fce6aefd41c7387"}],"number":315596}
buildkite-pr-comment-->

ashokaditya committed:
fix tests for 9.1

Not sure why the custom agent ids are not working on 9.1 branch.
This change here would differ from testing mocks in `8.19` and `main`

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/315728)
* Commit: 92ee813067c6471b68372116b92f4f08b91a0086

### Failed CI Steps
* [FTR Configs #101](https://buildkite.com/elastic/kibana-pull-request/builds/315728#0197d4b1-2336-405a-8d93-3bb0e0c54b18)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/315728#0197d4b1-2336-405a-8d93-3bb0e0c54b18) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/315728/jobs/0197d4b1-2336-405a-8d93-3bb0e0c54b18/artifacts/0197d4cf-2faa-4129-9eac-b65c3b30194a) FTR Configs #<span></span>101 / Cloud Security Posture Test adding Cloud Security Posture Integrations CSPM AWS CIS_AWS Organization Manual Direct Access CIS_AWS Organization Manual Direct Access Workflow

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #315596](https://buildkite.com/elastic/kibana-pull-request/builds/315596) failed d3cbdd63c5a6246031d9385de89b111e49bfa0f9
* :broken_heart: [Build #315515](https://buildkite.com/elastic/kibana-pull-request/builds/315515) failed 68fb09d1c2a1ad5badfc6ebb8fce6aefd41c7387

cc @ashokaditya

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/315728","number":315728,"commit":"92ee813067c6471b68372116b92f4f08b91a0086"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/315596","number":315596,"commit":"d3cbdd63c5a6246031d9385de89b111e49bfa0f9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/315515","number":315515,"commit":"68fb09d1c2a1ad5badfc6ebb8fce6aefd41c7387"}],"number":315728}
buildkite-pr-comment-->
